plots/plot.py

```diff
@@ -164,57 +180,63 @@ def plot_speedup():
     women_means = [1.8, 1.5, 1.1, 1.3, 0.9]
 
     x = np.arange(len(labels))  # the label locations
-    width = 0.35  # the width of the bars
+    width = 0.4  # the width of the bars
 
     fig, ax = plt.subplots()
-    rects1 = ax.bar(x - width / 2,
-                    men_means,
-                    width,
-                    label='Men',
-                    color=light_blue)
-    rects2 = ax.bar(x + width / 2,
-                    women_means,
-                    width,
-                    label='Women',
-                    color=dark_blue)
+    rects1 = ax.bar(x - width / 2, men_means, width, label='Men', color=light_blue)
+    rects2 = ax.bar(x + width / 2, women_means, width, label='Women', color=dark_blue)
 
     # Y-Axis Label
     #
     # Use a horizontal label for improved readability.
-    ax.set_ylabel('Speedup',
-                  rotation='horizontal',
-                  position=(1, 1.05),
-                  horizontalalignment='left',
-                  verticalalignment='bottom')
+    ax.set_ylabel(
+        'Speedup',
+        rotation='horizontal',
+        position=(1, 1.05),
+        horizontalalignment='left',
+        verticalalignment='bottom',
+    )
 
     # Add some text for labels, title and custom x-axis tick labels, etc.
     ax.set_xticks(x)
     ax.set_xticklabels(labels)
 
-    ax.legend(ncol=100,
-              loc='lower right',
-              bbox_to_anchor=(0, 1, 1, 0))
+    ax.legend(ncol=100, loc='lower right', bbox_to_anchor=(0, 1, 1, 0))
 
     autolabel(ax, rects1)
     autolabel(ax, rects2)
 
     save(fig, 'speedup.pdf')
 
 
+def csv_to_dict(file):
+    data = {}
+
+    if file is not None:
+        data = dict(csv.reader(file))
+
+    return data
+
+
 def main():
     parser = argparse.ArgumentParser(
         prog='plot',
         description='Plot the figures for this paper',
     )
     parser.add_argument('names', nargs='+', choices=['all', 'speedup'])
+    parser.add_argument('--layout', type=argparse.FileType('r'))
     args = parser.parse_args()
 
     setGlobalDefaults()
+    layout = csv_to_dict(args.layout)
 
     plotAll = 'all' in args.names
 
     if 'speedup' in args.names or plotAll:
-        plot_speedup()
+        with matplotlib.rc_context(
```

Comment:
This is where the plotting script configuration is applied from the exported CSV values.
If the file is not provided, we use sensible defaults.